e2etest/webhookstests/webhooks.go

```diff
@@ -381,3 +383,175 @@ var _ = ginkgo.Describe("Webhooks", func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("Webhooks", func() {
+	ginkgo.BeforeEach(func() {
+		ginkgo.By("Clearing any previous configuration")
+		err := ConfigUpdaterOtherNS.Clean()
```

Comment:
I'd expect there's no need for cleaning before or after, as all these tests won't add anything

Reply:
The latest version doesn't use setup code for namespace validation.